README.md

```diff
@@ -18,29 +18,11 @@ In addition to the object model, the SARIF SDK provides a set of helper classes
 ## Building the SDK
 
 If you want to build the SDK from source, rather than consuming the NuGet package,
```

Comment:
I didn't find this stuff in onboarding.  Readme is obviously good to have, but making it thinner with links into docs creates more paths to finding the content.  If you go to the Readme, you'll still find it via the link.  But if you're scanning the "docs" folder to see what's there, you'll now see "devbox-setup.md" whereas before you wouldn't see it at all.